tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorContent.spec.ts

```diff
@@ -0,0 +1,684 @@
+  ContentBuilder, PartialViewBuilder
+} from "@umbraco/json-models-builders";
+import {BlockListDataTypeBuilder} from "@umbraco/json-models-builders/dist/lib/builders/dataTypes";
+
+test.describe('BlockListEditorContent', () => {
+
+  const documentName = 'DocumentTestName'
+  const elementName = 'TestElement';
+  const blockListName = 'BlockListTest';
+
+  const documentAlias = 'documentTestName';
+  const elementAlias = 'testElement';
+  const blockListAlias = 'blockListTest';
+
+  test.beforeEach(async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.login();
+  });
+
+  async function createElement(umbracoApi, elementName) {
+    const elementType = new DocumentTypeBuilder()
+      .withName(elementName)
+      .withAlias(elementAlias)
+      .AsElementType()
+      .addGroup()
+        .withName("TestString")
+        .withAlias('testString')
+        .addTextBoxProperty()
+          .withLabel("Title")
+          .withAlias("title")
+        .done()
+      .done()
+      .build();
+    await umbracoApi.documentTypes.save(elementType);
+
+    return elementType;
+  }
+
+  async function createContentWithOneBlockListEditor(umbracoApi, documentName, documentAlias, elementName, elementAlias, blockListName, blockListAlias) {
+    const element = await createElement(umbracoApi, elementName);
+
+    const dataTypeBlockList = new BlockListDataTypeBuilder()
+      .withName(blockListName)
+      .addBlock()
+        .withContentElementTypeKey(element['key'])
+        .withSettingsElementTypeKey(element['key'])
+      .done()
+      .build();
+    const dataType = await umbracoApi.dataTypes.save(dataTypeBlockList);
+
+    const docType = new DocumentTypeBuilder()
+      .withName(documentName)
+      .withAlias(documentAlias)
+      .withAllowAsRoot(true)
+      .addGroup()
+        .withName('BlockListGroup')
+        .addCustomProperty(dataType['id'])
+          .withAlias(blockListAlias)
+        .done()
+      .done()
+      .build();
+    const generatedDocType = await umbracoApi.documentTypes.save(docType);
+
+    const rootContentNode = new ContentBuilder()
+      .withContentTypeAlias(generatedDocType["alias"])
+      .withAction("saveNew")
+      .addVariant()
+        .withName(blockListName)
+        .withSave(true)
+        .addProperty()
+          .withAlias(blockListAlias)
+          .addBlockListValue()
+            .addBlockListEntry()
+              .withContentTypeKey(element['key'])
+              .appendContentProperties(element.groups[0].properties[0].alias, "aliasTest")
+            .done()
+          .done()
+        .done()
+      .done()
+      .build()
+    await umbracoApi.content.save(rootContentNode);
+  }
+
+  test('can create content with a block list editor', async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
```

Comment:
Seems like this is done before and after each test, would love these in `beforeEach` & `afterEach` hooks instead to lower duplication 👍